lib/Event/OnLinkParsedEvent.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Event;
+
+use Doctrine\Common\EventArgs;
+
+final class OnLinkParsedEvent extends EventArgs
+{
+    public const ON_LINK_PARSED = 'onLinkParsed';
+
+    /** @var string */
+    private $url;
+    /** @var string */
+    private $linkType;
+    /** @var string */
+    private $currentFileName;
```

Comment:
I'm not positive about the Doctrine coding standards, but my guess is that the phpdoc above these is not needed/wanted

Reply:
They are required so phpstan, etc knows what type the properties are so we can enforce type consistency.

Reply:
We also need a new line inbetween each property.

Reply:
done ;)